Portfolio.html

```diff
@@ -18,7 +18,7 @@
       <header class="header-container">
```

Comment:
Please use this linter config. The project is missing `.stylelintrc.json` and `.hintrc` files.